Berry Big BoxStation Overhaul

BotTierPC opened the pull request:
## About The Pull Request

This PR is the culmination of my hyper-fixation created from the learning of Strong.dmm. It started out small, a simple wall removal, to a janitorial overhaul, followed by MetaStation Common Area Overhaul in #13359. And with it came a need to similarly change BoxStation's residential zone. This is where it got out of hand;

Beyond the departments themself I found my way to most parts of the station in some capacity, overhauling the Washrooms, Locker Room, Arrivals, Dorms, Lounge, Art Storage Room, while additionally adding in a run down in place of the moved Primary Tool Storage.

Essentially, this PR changes ALOT of non-departmental space on BoxStation in a Raspberry Flavored style.

## Why It's Good For The Game

Unlike alot of other PR's I dont believe the answer is as definitive and rather can fall to personal tastes as the entirety of this PR is visual changes to the layout of the station.

That said, most renovations here stemmed from issues I had personally taken with the map, and while it goes beyond that there are definitive benefits to this BoxStation-Alternative however minute. But beyond that, I personally like this PR, and the changes <sub> going to be honest, some parts of it need work, and is being worked on, but reviews are imporant and the PR is far enough along its time to publish and ask for </sub> because I feel like it brings it up to the modern standard that is being applied in other PR's, like the addition of Cardinal Station, and Rad Station's recent overhaul even if im heavily biased.

## Testing Photographs and Procedure

### In Round Screen-Shots;
 <details> <summary> Entrance Hallway</summary>

<img width="933" height="886" alt="Screenshot 2025-10-18 174951" src="https://github.com/user-attachments/assets/70038527-60e3-4fa9-bab9-3928601d57ab" />

</details>
 <details> <summary> Vacant Office</summary>

<img width="829" height="863" alt="Screenshot 2025-10-18 175156" src="https://github.com/user-attachments/assets/9d8928d4-ae43-4797-9620-b62364d867b4" />


</details>
 <details> <summary> Aux Tool Storage </summary>

<img width="693" height="483" alt="Screenshot 2025-10-18 175300" src="https://github.com/user-attachments/assets/af4275da-2be2-4a32-a424-4648fc943cfd" />


</details>
 <details> <summary> New Maintenance Casino </summary>

<img width="670" height="556" alt="Screenshot 2025-10-18 175233" src="https://github.com/user-attachments/assets/dd55358c-ba17-461d-97b7-2706b7b29699" />

</details>
 <details> <summary> Locker Room </summary>

<img width="946" height="773" alt="Screenshot 2025-10-18 175400" src="https://github.com/user-attachments/assets/b5b6cfcb-35f3-40c9-832c-e14fdbc25625" />


</details>

 <details> <summary> Locker Room Washroom </summary>

<img width="666" height="623" alt="Screenshot 2025-10-18 175416" src="https://github.com/user-attachments/assets/ac414f2a-f9a8-447b-93a2-b1bfc27cf198" />


</details>

 <details> <summary> Janitorial</summary>

<img width="867" height="409" alt="Screenshot 2025-10-18 230043" src="https://github.com/user-attachments/assets/07ba715d-420f-44c6-8411-dc359803ffaf" />
<img width="840" height="424" alt="Screenshot 2025-10-18 230050" src="https://github.com/user-attachments/assets/8a0f1e63-1c50-4ae5-8a2c-a4bb2877ab63" />


</details>

<details> <summary> Common Area </summary>

### Pool Area & Dorms
<img width="1024" height="635" alt="Screenshot 2025-10-18 230148" src="https://github.com/user-attachments/assets/b47af7f9-15c4-4e69-9eb9-b6fae4a61836" />
<img width="1103" height="644" alt="Screenshot 2025-10-18 230200" src="https://github.com/user-attachments/assets/e5fcf926-2c75-4d48-937e-01ae85885f37" />


### Lounge & Observation Room

<img width="728" height="784" alt="Screenshot 2025-10-18 230305" src="https://github.com/user-attachments/assets/581de1ed-86df-4565-b62a-aa7cfd2535be" />


</details>

<details> <summary> Fore Maintenance </summary>

<img width="492" height="444" alt="Screenshot 2025-10-18 230339" src="https://github.com/user-attachments/assets/a972e15e-e326-400c-9256-69fc5bd15750" />


</details>

<details> <summary> Art Storage Room </summary>

<img width="622" height="425" alt="Screenshot 2025-10-18 175428" src="https://github.com/user-attachments/assets/332145f6-fb8b-42e5-8ce6-209ebb3508a3" />

</details>

<details><summary>Lounge Washroom</summary>
<p>

<img width="617" height="646" alt="Screenshot 2025-10-18 230207" src="https://github.com/user-attachments/assets/70d2f6a4-fda4-4c72-96d1-c816cfd001bc" />
<img width="284" height="369" alt="Screenshot 2025-10-18 230213" src="https://github.com/user-attachments/assets/ce257caa-5f74-40be-a3b6-e7068a1da385" />

</p>
</details> 

<details><summary>Theatre</summary>
<p>

<img width="448" height="242" alt="Screenshot 2025-10-18 230438" src="https://github.com/user-attachments/assets/6b7e7d1a-4d1c-4472-bc10-3cb9ccee7248" />
<img width="585" height="574" alt="Screenshot 2025-10-18 230454" src="https://github.com/user-attachments/assets/72c9482c-1ef2-4821-a37c-b4059af311e3" />
<br> - In order to not have multiple things on the same wall tile, the beds had to be pushed together for the mirrors not to be but nothing else was touched beyond swapping the bed/desk.

</p>
</details> 

<details><summary>Arrivals Sec Outpost</summary>
<p>

<img width="998" height="745" alt="Screenshot 2025-10-18 175105" src="https://github.com/user-attachments/assets/375db851-70b2-4005-a34b-51e8a87fbaa7" />

</p>
</details> 

<details><summary>New Garden</summary>
<p>

<img width="873" height="516" alt="Screenshot 2025-10-18 175318" src="https://github.com/user-attachments/assets/18b340d3-af8e-4a07-9a69-aa3e0e321430" />


</p>
</details> 

### Strong.dmm Photos
<details> <summary> Everywhere! </summary>

<img width="824" height="539" alt="Screenshot 2025-10-18 231254" src="https://github.com/user-attachments/assets/a8d6ad23-896d-4a21-92e3-ec1105d064be" />
<img width="247" height="487" alt="Screenshot 2025-10-18 231300" src="https://github.com/user-attachments/assets/c0b6885b-7165-4092-bf80-b2e3a784cba7" />
<img width="603" height="458" alt="Screenshot 2025-10-18 231310" src="https://github.com/user-attachments/assets/473d4c5a-affe-4033-bbbe-314b33a80004" />
<img width="168" height="131" alt="Screenshot 2025-10-18 231317" src="https://github.com/user-attachments/assets/9f9d3744-1f15-4a9f-9009-aa897e689005" />


</details>

## Changelog
:cl:
tweak: Overhauls Non-Departmental Regions of BoxStation + Janitorial Department. Additionally adjusts and modernizes disposals to work with new changed areas.
/:cl:

BotTierPC committed:
Raspberian Boxstation Janitorial Hall Changes

Fixes the DANK and DIRTY hallway that boxstation arrivals>janitoral had, moving a sign and adding a place to sit instead of random tables

BotTierPC committed:
Update _basemap.dm

BotTierPC committed:
Update files.dm

BotTierPC committed:
Update BoxStation.dmm

BotTierPC committed:
Janitorial Rework

Removes the nearby unused storage room and adds the space into a revamped Janitorial

BotTierPC committed:
Removes not working decal

i tried to add a blood trail dyed black and renamed to look like oil tracks but its just not working as intended so gonzo with it (its already removed in the photo above)

BotTierPC committed:
Removes Black VV from Decal

A decal was dyed black, forgotten about and caused an error, prayers are that this fixes

BotTierPC committed:
Connets the APC to Grid

Adds 3 wires connecting the apc to wire line

BotTierPC committed:
Uses mapping_helpers/airlock... on 3 relevant

The nearby Maintence door, along with the two in janitorial get the helper

BotTierPC committed:
Fixes a error caused by pixels or something

No clue, literally had to turn it on and off again

BotTierPC committed:
Minor Changes based on suggested comments

Moved the event marker back INTO janitorial - Moves the light formerly on the wall with lightswitch/button to underneath the locker - Adds a second jani spawn location in the bay - removes decal under a wall

BotTierPC committed:
Merge branch 'RazPaTazMappin' into BoxedBerryBeautifully

BotTierPC committed:
The Entirety of Box Station Overhaul

Overhauls Arrivals, Primary+Aux Tool Storage, Garden, Locker Room, Janitorial, Dorms, Lounge, Vacant Office, Washrooms, Art Room, removes maint basketball court and adds maint casino.

BotTierPC committed:
Minor Fixes

Deletes some duplicate objectes, and a single tile of wrongful zoning more or less

BotTierPC committed:
Duplicat Obj Removal

Tables!!!!!!!!!!!!!!!!!!!

BotTierPC committed:
Disposals Routing Fix

Adds and changes disposals in the dorms area to NOT break everything and still function as intended hopefully

BotTierPC committed:
Disposals Fix

Changes the type of piping used for sorting in related locations and trys to fix the current issue of disposals not routing properly around security

BotTierPC committed:
Camera Move

Moves a camera in arrivals vacant office from the airtight flap over one to a wall tile

BotTierPC committed:
Changes Disposal Layout

I dont know what you want from me!!! IM JUST THROWING THINGS AGAINST THE WALL AT THIS POINT

BotTierPC committed:
Aforementioned Changes

BotTierPC committed:
Fixes window with no floor

BotTierPC committed:
Visual Changes to Cryo Room

BotTierPC committed:
WHOOPS

changes setting that i forgot to do first....

BotTierPC committed:
Locker Room Fixes

Makes the paper on the desk more relevant, and changes the washer section to be apart of the locker room area wise

Rukofamicom reviewed (dismissed):
Hard block on the public garden getting any of the following botany tools:

* Trays instead of dirt
* Botany chem dispenser
* Machinery dedicated to modifying plants in any way. 

The garden should not be remotely as functional as the actual botanical department.

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BoxedBerryBeautifully

BotTierPC committed:
Brings The PR To Ideal Order

ASSUMING IT WORKS!

BotTierPC committed:
Removes Tye-Dye Instructions and Adds Soap To Dorms Showers

BotTierPC committed:
Fixes Theatre room mirrors

BotTierPC committed:
Tiny stuff

so small not even going to mention here

BotTierPC committed:
Duplicate object removal

Deletes duplicate things lying around the map made from what i can only assume is updating the file

BotTierPC committed:
Virology Mail Connection

Routes the northern most disposal bin in virology to station network thus making it able to recieve mail

BotTierPC committed:
Tiny Stuff

Deletes some duplicate obj and matches all chairs in arrivals to be the same kind

BotTierPC committed:
Updates Disposals Entirely (still broken)

No more vv mail locations. Sec for sure is still broke though

BotTierPC committed:
Potentially Working Disposals?

MAYBE?

BotTierPC committed:
Directional Windows or something

BotTierPC committed:
Wires Abandonded Casino APC

BotTierPC committed:
actually hits the apc in casino

i overshot a tile

BotTierPC committed:
Final Update

Brings things up to match screenshots.  Barring any needed fixes/requested changes i am DONEZO

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BoxedBerryBeautifully

BotTierPC committed:
Adds Directional Windows To Arrival Holding Cel

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BoxedBerryBeautifully

BotTierPC committed:
Potential Mapping Conflict Resolution?

BotTierPC committed:
Aforementioned TM found changes

BotTierPC committed:
Fix stuff go

BotTierPC committed:
Uncooperative crayon gets gibbed

BotTierPC committed:
Revival of the crayon

BotTierPC committed:
TM Noticed Changes

Deletes a stray vendor in main hall, connects disposals to tool storage, adds a holocall to a few rooms, replaces the broke viewing section with laster tag closets, deletes toliet lights, fixes a missing pipe, and probably  something else i forgot

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BoxedBerryBeautifully

BotTierPC committed:
Merge remote-tracking branch 'upstream/master' into BoxedBerryBeautifully
